userguide/system.rst

```diff
@@ -822,6 +818,108 @@ with locking enabled:
        LockOnReset:     1
 
 
+.. index:: SED, SED Password
+.. _Managing SED Password and Data:
+
+Managing SED Password and Data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This section contains command line instructions to manage SED
+passwords and data. The command used is
+`sedutil-cli(8) <https://www.mankier.com/8/sedutil-cli>`__. Most
+SEDs are TCG-E (Enterprise) or TCG-Opal
```

Comment:
Not clear why this sentence is here. As the reader, why do I need to know this?  *Do* I need to know this?

Reply:
I would say they need to know this because there are different instructions for each.

Reply:
Okay, so it should explain that after that sentence.
`Commands are different for the different drive types, so the first step is identifying which type is being used.`